dirs/dirs.go

```diff
@@ -190,6 +198,74 @@ func init() {
 	SetRootDir(root)
 }
 
+// SnapHomeDirs returns a slice of the home directories currently in use.
+func SnapHomeDirs() []string {
+	snapHomeDirsMu.Lock()
+	defer snapHomeDirsMu.Unlock()
+	dirs := make([]string, len(snapHomeDirs))
+	copy(dirs, snapHomeDirs)
+	// Should never be true since SetSnapHomeDirs is ran on init and on SetRootDir calls.
+	// Useful for unit tests.
+	if len(dirs) == 0 {
+		return []string{filepath.Join(GlobalRootDir, "/home")}
+	}
+	return dirs
+}
+
+// SetSnapHomeDirs sets SnapHomeDirs to the user defined values and appends /home if needed.
+// Homedir must be a comma separated list of the user defined home directories.
+// If homedirs is empty, SnapHomeDirs will be a slice of length 1 containing "/home".
+// Also generates the data directory globbing expressions for each user.
+// Ran at startup by configstate.Init, returns the split outsnap home directories. Use DataHomeGlobs to retrieve globs.
```

Comment:
This last sentence is confusing and I don't know what this means except it's run on startup by configstate.Init, could you please rewrite this?

Reply:
how about `//Ran at startup by configstate.Init, returns a slice of home directories.`?

Reply:
Sounds better, but maybe `Expected to be run by ...`